src/components/Verify.js

```diff
@@ -446,13 +454,22 @@ const Verify = (props) => {
             <Navbar
               buttons={[
                 <Button
-                  variant="text"
+                  variant="outlined"
                   color="primary"
                   onClick={handleFilterClick}
                   startIcon={<IconFilter />}
                   key={1}
                 >
                   Filter
+                  {
+                    countAppliedFilters(verifyContext.filter) > 0 ?
+                      <Avatar className={classes.activeFilters}>
+                      {
+                        countAppliedFilters(verifyContext.filter)
```

Comment:
There's no need to call the function twice.
Assign the result to a variable and check that instead.